src/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+
+const LinkView = ({ targetTo, text, path }) =>
+  (<a
+    tabIndex="0"
+    role="link"
+    onClick={(event) => {
```

Comment:
We might want to add more properties later (like or `class`, `title`, `target` or even the fancy new `download`). To support this, we can add `...props` to `LinkView` function arguments and add `{...props}` to the element.